.circleci/config.yml

```diff
@@ -1,4 +1,38 @@
-version: 2
+version: 2.1
+workflows:
+  pr_acceptance:
+    jobs:
+      - test
+      - build
+  build_accept_deploy:
+    jobs:
+      - test
+      - build
+      - package:
+          requires:
+            - build
+      - deploy:
+          requires:
+            - test
+            - build
+            - package
+  publish_release:
+    jobs:
+      - test:
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /v[0-9]+(\.[0-9]+)*(-.*)*/
+      - github_binaries:
+          requires:
+            - test
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /v[0-9]+(\.[0-9]+)*(-.*)*/
```

Comment:
Had to repeat the filtering in both jobs because there is no way to filter an entire workflow